resources/prosody-plugins/mod_muc_lobby_rooms.lua

```diff
@@ -373,6 +377,23 @@ function update_session(event)
     end
 end
 
+function handle_create_lobby(event)
```

Comment:
Can you change the 'muc-config-submitted' hook to use this method, so we avoid code duplication?
Maybe make the function return true/false so you can act whether lobby room was created or not (the notification and the `event.status_codes['104'] = true;` code)

Reply:
Also do the save room only if room was created.

Reply:
Done.